### ISSUE TITLE
`TimerTest::testLongExecutionTime` failed

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/CodeIgniter4#6823]
```
 1) CodeIgniter\Debug\TimerTest::testLongExecutionTime
Failed asserting that 3600 is equal to 1 or is less than 1.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Test/CIUnitTestCase.php:436
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Debug/TimerTest.php:98
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:97
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/3402855272/jobs/5660114944

**Comment by kenjis** [comment]
```php
        d(strtotime('-110 minutes')); // 1667713787
        d(strtotime('now'));          // 1667723987
```

**Comment by kenjis** [comment]
I do not know if this is a bug or not. But it is not intuitive that this test fails.
https://github.com/codeigniter4/CodeIgniter4/blob/4a748b935e7594d2e7b1dedd6dab800dd3f96b1e/tests/system/Debug/TimerTest.php#L94-L99

**Comment by paulbalandan** [comment]
Fixed by #6826
